views/signup.handlebars

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous" defer></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous" defer></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous" defer></script>
+    <title>Register</title>
+
+    <link rel = "stylesheet"
+   type = "text/css"
+   href = "../css/signup_style.css"/>
+  </head>
+  <body>
+
+	<div class="container">
+	    <div class="row">
+	        <div class="sign_in_form">
+	            <form action="/signup/create_user" method="POST">
+	                <h4 class="sign_in_message" align="center">Sign Up for YetiEats!</h4>
+ 	                <span class="align-center">Already registered? <a href="/signin"/>Click here to login</a></span>
+	                <br/>
+	                <div class="form-group">
+	                    <input type="text" class="form-control" name="firstname" placeholder="First Name" required>
+	                </div>
+	                <div class="form-group">
+	                    <input type="text" class="form-control" name="lastname" placeholder="Last Name" required>
+	                </div>
+	                <div class="form-group">
+	                    <input type="email" class="form-control" name="email" placeholder="Email" required>
+	                </div>
+	                <div class="form-group">
+	                    <input type="text" class="form-control" name="username" placeholder="User Name" required>
+	                </div>
+	                <div class="form-group">
+	                    <input type="password" class="form-control" name="password" placeholder="Password" required>
```

Comment:
should we also have the user confirm their password?

Reply:
would you kindly elaborate?

Reply:
Meaning have a second input form for the user to re-enter their password.